lang/hr.js

```diff
@@ -150,7 +153,7 @@ var language = {
       },
       "command-flash": {
         "title": "Učitaj (\"flash\") projekt izravno u micro:bit",
-        "label": "Flash"
+        "label": "\"Flashaj\""
```

Comment:
Why are there apostrophes around the word? `"Flashaj"`

Reply:
This is how it's been translated and approved. I have [asked in Crowdin](https://crowdin.com/translate/microbitorg/3298/en-hr#q=flashing) whether this is required

Reply:
Thanks!

Reply:
Was this resolved?

Reply:
Yes, the answer they gave was that if a word has been co-opted into the language as they do not have an equivalent then they use `""` around the word

Reply:
To be honest I am not a fan of that, for most english speaking people "flash" doesn't mean the same thing, and would likely use quotes when included in a sentence like this one. But the quotes shouldn't be added to the buttons.